pkg/cmd/registry/artifact/crud/get/get.go

```diff
@@ -95,7 +95,7 @@ rhoas service-registry artifact get --artifact-id=myartifact --version=4
 	cmd.Flags().StringVarP(&opts.artifact, "artifact-id", "a", "", "Id of the artifact")
 	cmd.Flags().StringVarP(&opts.group, "group", "g", util.DefaultArtifactGroup, "Artifact group")
 	cmd.Flags().StringVar(&opts.registryID, "instance-id", "", "Id of the registry to be used. By default uses currently selected registry")
-	cmd.Flags().StringVar(&opts.outputFile, "file-location", "", "Location of the output file")
+	cmd.Flags().StringVar(&opts.outputFile, "file", "", "Location of the output file")
```

Comment:
My assumption is that you meant to have it as others?
```suggestion
	cmd.Flags().StringVar(&opts.outputFile, "output-file", "", "Location of the output file")
```